adobe/spectrum-web-components
# test: ensure Lit Dev Mode warnings are not emitted

## Description
Test default package consumption patterns to ensure they do not emit Lit Dev Mode warnings.

## Related issue(s)
- refs #2257 

## Motivation and context
Safe guard the return of possibly perf negative patterns.

## How has this been tested?

    1. Tests run
    2. No failures
    1. Undo the changes in https://github.com/adobe/spectrum-web-components/commit/b2c393ef5da68cb832cbf6cddb1b96c67423fb80
    2. Run the `action-button` units tests
    3. See them fail

## Types of changes

## Checklist
